routes/web.php

```diff
@@ -98,4 +91,5 @@
 /*-----------------------*
  * Site Controller *
  *-----------------------*/
+Route::get('site/{site}/locations', 'SiteController@locations');
```

Comment:
There is an eloquent relation for this in the site model. Why is a new route needed?